migrations/00086_study_topics.sql

```diff
@@ -0,0 +1,2219 @@
+-- +goose Up
```

Comment:
Just a bunch of table generation, shouldn't be any breaking table edits or anything in here. Some sorting, and some hiding of existing things, but nothing scary.